tests/test_visualization.py

```diff
@@ -52,11 +130,12 @@ def fail_objective(_):
     assert len(figure.data) == 0
 
 
-def test_get_optimization_history_plot():
-    # type: () -> None
+@pytest.mark.parametrize('direction', ['minimize', 'maximize'])
+def test_get_optimization_history_plot(direction):  # type: ignore
```

Comment:
It seems that `# type: ignore` isn't required here:
```suggestion
def test_get_optimization_history_plot(direction): 
```

Reply:
thanks.